services/src/unit-test/java/uk/gov/moj/sdt/services/mbeans/SdtManagementMBeanTest.java

```diff
@@ -114,207 +144,178 @@ public class SdtManagementMBeanTest extends AbstractSdtUnitTestBase {
+    void setMdbPoolSizePoolSizeInvalid() {
         final DefaultMessageListenerContainer messageListenerContainer = new DefaultMessageListenerContainer();
         messageListenerContainer.setDestinationName(TEST_QUEUE_NAME);
 
         this.sdtManagementMBean.setMessageListenerContainer(messageListenerContainer);
 
         String returnVal = this.sdtManagementMBean.setMdbPoolSize(TEST_QUEUE_NAME, TEST_POOL_SIZE + 41);
 
-        Assert.assertEquals("The return message should have been an error",
-                "MDB pool size can only be set between 1 and 50.", returnVal);
+        assertEquals("MDB pool size can only be set between 1 and 50.", returnVal,
+                RETURN_MESSAGE_SHOULD_HAVE_BEEN_AN_ERROR);
 
         returnVal = this.sdtManagementMBean.setMdbPoolSize(TEST_QUEUE_NAME, TEST_POOL_SIZE - 10);
 
-        Assert.assertEquals("The return message should have been an error",
-                "MDB pool size can only be set between 1 and 50.", returnVal);
+        assertEquals("MDB pool size can only be set between 1 and 50.", returnVal,
+                RETURN_MESSAGE_SHOULD_HAVE_BEEN_AN_ERROR);
     }
 
     /**
      * Test method for setMdbPoolSize. A test for the positive conditions scenario.
      */
     @Test
-    public void setMdbPoolSize() {
+    void setMdbPoolSize() {
         final DefaultMessageListenerContainer messageListenerContainer = new DefaultMessageListenerContainer();
         messageListenerContainer.setDestinationName(TEST_QUEUE_NAME);
 
         this.sdtManagementMBean.setMessageListenerContainer(messageListenerContainer);
 
         final String returnVal = this.sdtManagementMBean.setMdbPoolSize(TEST_QUEUE_NAME, TEST_POOL_SIZE);
 
-        Assert.assertEquals("The return message is incorrect", "mdb pool [" + TEST_QUEUE_NAME +
-                "] size changed from 1 to " + TEST_POOL_SIZE, returnVal);
-
-        Assert.assertEquals("The message listener container max concurrent pool size should have been changed", 10,
-                messageListenerContainer.getMaxConcurrentConsumers());
+        assertEquals("mdb pool [" + TEST_QUEUE_NAME + "] size changed from 1 to " + TEST_POOL_SIZE, returnVal,
+                "The return message is incorrect");
 
+        assertEquals(10, messageListenerContainer.getMaxConcurrentConsumers(),
+                "The message listener container max concurrent pool size should have been changed");
     }
 
     /**
      * Test method for requeueOldIndividualRequests. This method tests the scenario
      * where there are no old individual requests to process.
      */
     @Test
-    public void requeueOldRequestsNotFound() {
-        final List<IIndividualRequest> individualRequests = new ArrayList<IIndividualRequest>();
-        EasyMock.expect(mockIndividualRequestDao.getStaleIndividualRequests(TEST_STALE_DURATION)).andReturn(
-                individualRequests);
-
-        EasyMock.replay(mockIndividualRequestDao);
+    void requeueOldRequestsNotFound() {
```

Comment:
I don't think this test actually proves that the requests were not re-queued.  Probably needs to check that methods on the mocked messagingUtility and individualRequestDao classes were not invoked.  Should be able to do this using the Mockito never() method./

Reply:
done, called times(0) on methods mockIndividualRequestDao.persistBulk and mockMessagingUtility.enqueueRequest

Reply:
Confirmed that test now verifies that methods on mocked classes are not invoked.